app/data/mwss_4.json

```diff
@@ -242,7 +243,7 @@
                                     "description": "",
                                     "id": "weekly-pay-question-6",
                                     "number": "2.3",
-                                    "title": "Of the [TOTAL gross PAY] paid to <em>weekly paid</em> employees in the <em>last week of December 2016</em>, what amount was:",
+                                    "title": "Of the {{answers.weekly_total_gross}} paid to <em>weekly paid</em> employees in the <em>last week of December 2016</em>, what amount was:",
```

Comment:
I think there should be a pound sign in this text, it currently looks like this:
![image](https://cloud.githubusercontent.com/assets/1754822/22520184/3e5106b6-e8ab-11e6-9579-5030979b5dbf.png)

Same for all other occurrences.